persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/DsePersistence.java

```diff
@@ -188,11 +190,15 @@ protected void initializePersistence(Config config) {
 
     waitForSchema(STARTUP_DELAY_MS);
 
-    if (USE_PROXY_PROTOCOL) interceptor = new ProxyProtocolQueryInterceptor();
-    else interceptor = new DefaultQueryInterceptor();
+    if (USE_PROXY_PROTOCOL) {
+      interceptor = new ProxyProtocolQueryInterceptor();
+    } else {
+      interceptor = new DefaultQueryInterceptor();
+    }
```

Comment:
This will likely conflict with recent changes in another PR from @mpenick

Reply:
Didn't notice. That was actually the formatter's doing.